Page about staking from the consumer perspective (#493)
trilitech/tezos-developer-docs: docs/using/staking.md

---
title: Staking
authors: "Tim McMackin"
last_update:
  date: 17 December 2024
---

> Staking is the process of temporarily locking tez on the Tezos platform.

Reviewer: State right from the beginning the usefulness for yourself:
```suggestion
Staking is the process of temporarily locking tez on the Tezos platform in exchange of rewards.
```

Author: I made this "in exchange for rewards".

> Staked tez stays in the staker's account, but they cannot transfer that tez while it is staked.

Reviewer: shouldn't it be plural? "the tez staked" suggests 1 tez, but maybe I'm wrong:
```suggestion
Staked tez stays in the staker's account, but they cannot be spent (e.g., transferred to another account).
```

Author: It's tricky, "tez" can be singular or plural. I made this "but the staker cannot transfer (or spend) that tez."

> They can unstake the tez and make it spendable after a delay.

Reviewer: ```suggestion
Stakers can unstake the tez at any time, which makes them spendable again after a delay.
```

> Staking is an important part of keeping Tezos secure and earning rewards from it.

Reviewer: ```suggestion
Staking is an important part of running the Tezos protocol and keeping the blockchain secure, so this is why stakers earn rewards proportional to the locked funds.
```

Two main groups stake on Tezos:

> - Bakers must stake 6,000 tez to receive the right to bake blocks and to receive rewards for baking those blocks.

Reviewer: Don't assume end users know what is a baker:
```suggestion
- Bakers, the creators of new blocks in Tezos, must stake 6,000 tez to receive the right to "bake" blocks and to receive rewards for baking those blocks.
```

Their staked tez ensures that they bake correctly, because part of their stake is taken ("slashed") if they misbehave.
For more information about staking for baking purposes, see [Baking](/architecture/bakers).

> - Any Tezos user can stake any amount of tez with a baker and earn rewards based on how much they stake.

Reviewer: de-duplicate "any amount" and "based on how much", which are detailed just after:
```suggestion
- Any Tezos user can stake tez with a baker and earn rewards for them.
```

In exchange for staking tez with a baker, users automatically receive a portion of the baker's rewards in proportion to how much they stake.
Users can stake any amount of tez, but there is a limit to how much staked tez a single baker can accept, and bakers must opt in to allow users to stake with them.

Staking has some similarities to the concept of a Smart Rollup bond, but there are important differences; see [Bonds](/architecture/smart-rollups#bonds).

## Staking as a Tezos user

> The process of user staking has these main steps:

Reviewer: "user staking" sounds weird, it's like you stake the user, 😄 , can we say:
```suggestion
The process of staking has these main steps for a user:
```

1. The user selects a baker as a delegate for their account.
As described in [Delegating to a baker](/architecture/bakers#delegating-to-a-baker), delegating tez to a baker puts that tez toward the baker's voting and baking power.
Delegating incurs no risk to the user; the user retains full control of the tez, they can spend it at any time, and they are not punished if the baker misbehaves.

1. The user stakes tez with the baker.

> Staking the tez locks it temporarily, but the tez stays in the user's account.

Reviewer: (it's really a question, I just want to learn:)
so ok, "tez" can be singular or plural, but in this case it *is* plural, so don't we have to agree in number?
```suggestion
Staking the tez locks them temporarily, but the tez stay in the user's account.
```

However, if the baker misbehaves, their punishment also affects tez staked with them, so it's important to choose a responsible baker.

1. The user leaves their tez staked for as long as they want.
During this time the rewards are added to their account automatically.

> THe process of unstaking has these main steps:

Reviewer: ```suggestion
The process of unstaking has these main steps:
```

1. When the user wants to stop staking, they decrease the amount that they have staked or unstake all of their staked tez.

1. After a delay of about 10 days, the tez are unfrozen in their account and the user can use that tez as usual.

## How to stake

The easiest way to stake is to use the staking web application at https://stake.tezos.com, which walks you through the process.

:::warning

> Make sure that you are staking at the right URL (stake.tezos.com).

Reviewer: just for security:
```suggestion
Make sure that you are staking at the right URL (https://stake.tezos.com).
```
